libraries/AP_NavEKF3/AP_NavEKF3_MagFusion.cpp

```diff
@@ -462,6 +475,20 @@ void NavEKF3_core::SelectMagFusion()
     }
 }
 
+#if HAL_LOGGING_ENABLED
+// log changes to magnetometer fusion selection
+void NavEKF3_core::WriteMagFusionSelection()
+{
+    const struct log_XKMF pkt{
```

Comment:
quite an expensive way to log 1 byte, could be a field in XKFS, or could jst document how to tell using XKF3.IYAW vs XKF3.IMX/IMY/IMZ